packages/arb-bridge-eth/contracts/validator/GasRefunder.sol

```diff
@@ -27,6 +27,9 @@ contract GasRefunder is IGasRefunder, Ownable {
     mapping(address => uint256) public lastContractRefund;
     uint256 public maxRefundeeBalance;
     uint256 public extraGasMargin;
+    uint256 public calldataCost;
```

Comment:
Do we need uint256 for each of these? Can half the storage slots by using uint128

Reply:
Also I'd prob use a struct to get set these